com.inklestudios.Inky.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "com.inklestudios.Inky",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "20.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "inky",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=dri",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "inky",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/inky",
+                "mv -t /app/inky *"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/manuq/inky/releases/download/alpha/Inky-linux-x64.zip",
+                    "sha256": "e6150f1d999a83a4a4ed7af478366136540f361fbe85f09c21dacc01d2e8b380",
+                    "only-arches": [
+                        "x86_64"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "wrapper",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install inky-wrapper.sh /app/bin/inky",
+                "install -Dm644 com.inklestudios.Inky.appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml",
+                "install -Dm644 inky.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png",
+                "install -Dm644 com.inklestudios.Inky.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "inky-wrapper.sh",
+                    "commands": [
+                        "exec /app/inky/Inky \"$@\""
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "com.inklestudios.Inky.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Thanks for your review. You mean the file suffix should be `.metainfo.xml`? Or is there anything to change in the format?

Reply:
.metainfo.xml

Reply:
Thanks for clarifyng. I just pushed the rename.